From 8bc220189a687291639f81ea6378aaba26e4edd7 Mon Sep 17 00:00:00 2001
From: hasufell <hasufell@posteo.de>
Date: Sun, 22 Dec 2013 16:57:19 +0100
Subject: [PATCH] fix Version entry in lutris.desktop

Version refers to the spec version, not the program version
http://standards.freedesktop.org/desktop-entry-spec/latest/ar01s05.html
---
 lutris.desktop    | 2 +-
 lutris.desktop.in | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lutris.desktop b/lutris.desktop
index 34dbb3f..e022382 100644
--- a/lutris.desktop
+++ b/lutris.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=0.3.2
+Version=1.0
 Name=Lutris
 Comment=Lutris application
 Categories=Network;FileTransfer;Game;
diff --git a/lutris.desktop.in b/lutris.desktop.in
index 6695909..4daa576 100644
--- a/lutris.desktop.in
+++ b/lutris.desktop.in
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=0.3.2
+Version=1.0
 Name=Lutris
 Comment=Lutris application
 Categories=Game;
-- 
1.8.5.1
